infra/k8s/kops-weave/s3bucket.yml

```diff
@@ -0,0 +1,46 @@
+apiVersion: apps/v1
+kind: DaemonSet
```

Comment:
Actually this can replace the `dummy` daemonset, so I will probably unite them in a future PR.

Reply:
Already removed `dummy` daemonset from the README, `s3bucket` now does the function to create an `cni0` interface to be detected by the `sidecar`.